package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "react-native-permissions",
-  "version": "3.1.0",
+  "version": "3.2.0",
```

```suggestion
  "version": "3.1.0",
```

bumping package versions is typically the domain of the maintainer (and frequently fully automated) - I'd remove this - as a maintainer on other packages I will never merge a version change in a PR personally

---

Sorry it's my first time collaborating in a library and I didn't know it. I'll change

---

no worries, most people never even make that first try. There are lots of subtle things like that, but in the end - forward progress!